styles.scss

```diff
@@ -0,0 +1,283 @@
+            @include mixins.flex(column);
+            text-align: center;
+            width: 80%;
+            height: 70vh;
+            margin: auto;
+        }
+        &__experience{
+            font-size: 1.1rem;
+            padding: 0px
+        }
+        &__education{
+            @include mixins.about-card;
+            font-size: 1.1rem
+        }
+
+        &__text {
+            width: 100%;
+            display: flex;
+            flex-direction: column;
+            div {
+                img {
+                    height: 250px;
+                    width: 250px;
+                }
+            }
+        }
+    }
+    .skills {
+        padding-bottom: 100px;
+        h2 {
+            @include mixins.sub-headings-mobile;
+        }
+    }
+    .projects {
+        h2 {
+            @include mixins.sub-headings-mobile;
+        }
+        &__container {
+            @include mixins.flex(column);
+        }
+        &__card {
+            width: 80%;
+            &__heading {
+                a {
+                    @include mixins.button;
+                    margin: 10px;
+                }
+            }
+        }
+    }
+    .contact {
+        h2 {
+            @include mixins.sub-headings-mobile;
+        }
+        &__links {
+            flex-direction: column;
+            gap: 25px;
+        }
+    }
+    .footer {
+        text-align: center;
+    }
+}
+
+@include mixins.mobile {
+    .about {
+        h2 {
+            font-size: 1.4rem;
+            padding: 80px 0;
+        }
+        &__container {
+            width: 100%;
+        }
+        &__experience{
+            width: 60%;
+        }
+        &__education{
+            width: 60%;
+            font-size: 1.1rem;
+            padding: 0px;
+            margin-bottom: 20px;
+        }
+        &__text {
+            div {
+                width: 80%;
+                padding: 20px;
+                img {
+                    height: 250px;
+                    width: 250px;
+                }
+            }
+        }
+    }
+    .skills {
+        h2 {
+            font-size: 1.4rem;
+            padding: 80px 0;
+        }
+        &__container {
+            margin-bottom: 10px;
+            div {
+                font-size: 1.2rem;
+                margin: 5px;
+            }
+            img {
+                height: 60px;
+                width: 60px;
+            }
+        }
+    }
+    .projects {
+        h2 {
+            font-size: 1.4rem;
+            padding: 80px 0;
+        }
+        img {
+            width: 250px;
+            height: 150px
+        }
+    }
+    .contact {
+        h2 {
+            font-size: 1.3rem;
+            padding: 80px 0;
+        }
+    }
+    .footer {
+        margin: 4px;
+        text-align: center;
+        nav {
+            text-align: center;
+            a {
+                padding: 0px 5px;
+                margin: 10px;
+                text-align: center;
+            }
+        }
+    }
+}
```

Comment:
I'd rather have the mixin used in each individual section than have all the overrides here in styles.scss. Yeah it is a few more media queries but the readability gain over the performance is worth it for me.